src/internal.c

```diff
@@ -7109,8 +7112,14 @@ int SendKexDhReply(WOLFSSH* ssh)
                 else {
                     WLOG(WS_LOG_INFO, "Signing hash with %s.",
                             IdToName(ssh->handshake->pubKeyId));
+#ifdef WOLFSSH_TPM
+                    wolfTPM2_SignHashScheme(ssh->ctx->tpmDev, ssh->ctx->tpmKey,
```

Comment:
I don't see any return code checking on this call? What if there is a TPM failure? What if `ssh->ctx->tpmDev` or `ssh->ctx->tpmKey` are not set?